### ISSUE TITLE
Can't remove dependency when local registry removed

### ISSUE THREAD

**Issue by isaced** [jakeheis/Ice#22]
I think it should be possible to remove the dependency at this time?

1. `ice registry add https://github.com/xxx/xxx xxx`
2. `ice add xxx`
3. `ice registry remove xxx`
4. `ice remove xxx`

    > Error: https://github.com/xxx has no manifest for version 0.1.0

**Comment by jakeheis** [comment]
Perhaps I'm misunderstanding the issue, but this worked fine for me:

```shell
> ice registry add https://github.com/jakeheis/Icebox Icebox
> ice add Icebox
Fetch https://github.com/jakeheis/Icebox
...
> ice registry remove Icebox
> ice remove Icebox
```

Could you explain in a little more detail what issue you're having?

**Comment by isaced** [comment]
The problem is that my repository has no manifest (for test), then I can't remove it (ice remove xxx).

**Comment by jakeheis** [comment]
Fixed by 7f837ad
